test/controllers/organizations_controller_test.rb

```diff
@@ -0,0 +1,92 @@
+require 'test_helper'
+
+class OrganizationsControllerTest < ActionController::TestCase
+  def setup
+    @user             = users(:tobias)
+    session[:user_id] = @user.id
+  end
+
+  test '#new returns success' do
+    stub_get_json("https://api.github.com/users/#{@user.login}/orgs",
```

Comment:
This is my first stab at this kind of testing, I definitely want to DRY this up though as there is a lot of repeated code. But overall what do you think?